internal/handler/web/main.go

```diff
@@ -24,3 +28,10 @@ func New() *Handler {
 		auth:       auth.New(),
 	}
 }
+
+func (h *Handler) GetLogout(ctx *gin.Context) {
+
+	logoutUrl := "/oauth2/sign_out?rd=https://login.microsoftonline.com/" + config.EntraCredentials().TenantID + "/oauth2/v2.0/logout?post_logout_redirect_uri=" + config.EntraInviteRedirectURL()
+
+	ctx.Redirect(http.StatusFound, logoutUrl)
+}
```

Comment:
Could this equally work as a direct link from the frontend? For login we currently have:

```
      <Button className={styles.button} size="large" href={`/oauth2/start?`} cy="login">
        Login with UCL SSO
      </Button>
```

I don't feel strongly about it one way or the other. Unless the implementation is simpler via a direct link?

Reply:
I see, it would be nice if there was a way to utilise something like `href={`/oauth2/logout`}` but fair enough if it's not possible.